CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.36.0
+* Feature - structured logs and logfile rollover features [#2311](https://github.com/aws/amazon-ecs-agent/pull/2311) and [#2319](https://github.com/aws/amazon-ecs-agent/pull/2319)
+* Feature - new ECS_LOG_* env vars [#2330](https://github.com/aws/amazon-ecs-agent/pull/2330)
+* Feature - new ECS_HOST_DATA_DIR and ECS_DATA_DIR env vars [#2328](https://github.com/aws/amazon-ecs-agent/pull/2328)
```

Comment:
ECS_HOST_DATA_DIR and ECS_DATA_DIR are not new and it's just the doc is updated.. i don't think we need to mention this in readme

Reply:
done